posthog/queries/test/test_lifecycle.py

```diff
@@ -607,17 +553,12 @@ def test_filter_test_accounts(self):
                 request,
             )
 
-    return TestLifecycle
-
-
-def _create_action(**kwargs):
-    team = kwargs.pop("team")
-    name = kwargs.pop("name")
-    action = Action.objects.create(team=team, name=name)
-    ActionStep.objects.create(action=action, event=name)
-    action.calculate_events()
-    return action
+        def assertLifecycleResults(self, results, expected):
+            sorted_results = [
+                {"status": r["status"], "data": r["data"]} for r in sorted(results, key=lambda r: r["status"])
+            ]
+            sorted_expected = list(sorted(expected, key=lambda r: r["status"]))
 
+            self.assertEquals(sorted_results, sorted_expected)
 
-class TestDjangoLifecycle(lifecycle_test_factory(Trends, Event.objects.create, Person.objects.create, _create_action)):  # type: ignore
```

Comment:
non-blocking

Where did these go?

Reply:
They started failing and given we don't support postgres anymore yeeted them out of here. There's other cleanup to do but wanted to minimize the diff to keep it easier to review. :)